README.md

```diff
@@ -1,6 +1,6 @@
 # BYU Web Community CDN
```

Comment:
I'm assuming you updated the readme to match your experience. I wasn't involved enough to know if this all matches that.

Reply:
Yes, I at least tried to correct everything incorrect that I could find